src/main/java/org/skriptlang/skript/misc/colours/ExprHex.java

```diff
@@ -0,0 +1,27 @@
+package org.skriptlang.skript.misc.colours;
+
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.util.Color;
+import org.jetbrains.annotations.Nullable;
+
+public class ExprHex extends SimplePropertyExpression<Color, String> {
+
+	static {
+		register(ExprHex.class, String.class, "hex [code]", "colors");
+	}
+
+	@Override
+	public @Nullable String convert(Color from) {
+		return ColourUtils.toHex(from);
+	}
+
+	@Override
+	protected String getPropertyName() {
+		return "hex code";
+	}
+
+	@Override
+	public Class<? extends String> getReturnType() {
+		return String.class;
+	}
```

Comment:
I'm not sure about this
I think i'd rather see it implemented as taking in a number and returning a string, rather than being restricted to colors